proc/traverse/over.go

```diff
@@ -0,0 +1,97 @@
+package traverse
```

Comment:
Do you want to change the package name and directory name from travers to over?

Reply:
No, because over/into both belong in traverse.  This will get rejiggered in the next step.